documentation/slate/source/includes/nearby.md

```diff
@@ -25,32 +19,17 @@ object, and the requested object.
 | `get` /coverage/*lon;lat*/*collection_name*             | Collection of objects of a region   |
 | `get` /coverage/*lon;lat*/*collection_name*/*object_id* | Information about a specific region |
 
--   **[Journeys](#journeys)** : Compute journeys
-
-| url | Result |
-|------------------------------------------|-------------------------------------|
-| `get` /coverage/*resource_path*/journeys | List of journeys or isochrones      |
-| `get` /journeys                          | List of journeys or isochrones      |
-
-
--   **[Route Schedules](#route-schedules)**, **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** : 
+-   **[Stop Schedules](#stop-schedules)**, **[Departures](#departures)** and **[Arrivals](#arrivals)** : 
 Compute time tables for a given resource
 
 | url | Result |
 |-------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
-| `get` /coverage/*resource_path*/route_schedules | List of the entire route schedules for a given resource                                                       |
 | `get` /coverage/*resource_path*/stop_schedules  | List of the stop schedules grouped by ``stop_point/route`` for a given resource                               |
+| `get` /coverage/*lon;lat*/stop_schedules        | List of the stop schedules grouped by ``stop_point/route`` for coordinates                                    |
```

Comment:
some more precisions on the differences with the previous line would be nice (also for lines 30 and 32)